### PR TITLE
Profile feature

### PR DESCRIPTION
**Pull request by michael-smith-nz**
@ahdinosaur Review away. More to come :smile:

### CODE REVIEW

File: api/migrations/20160219164214_create_profile.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
+
+exports.up = function(knex, Promise) {
+    return knex.schema.createTableIfNotExists('profiles', function(table) {
+      table.increments('id')
+      table.string('name')
+      table.string('note')
+      table.string('image')
+  })
+  .then(function() {
+    return knex.insert({name:'Jane', note: 'Loves... things', image: '../profile.jpg'})
```
Comment (ahdinosaur) [inline api/migrations/20160219164214_create_profile.js]: these should be within a [knex seed file](http://knexjs.org/#Seeds-CLI)

File: app/highlights/components/newHighlightList.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+}
+
+function render ({ props, children }) {
+  return h('div', {id: 'newHighlightList'}, map(children, (child, childKey) => {
+    return h(child, {}, [])
+  }),
+  h('button', {onClick: addHighlight }, 'Add new highlight'))
+}
+function addHighlight() {
+  //How to add a component I don't know about?
```
Comment (ahdinosaur) [inline app/highlights/components/newHighlightList.js]: you could add a new highlight by dispatching a change in internal component state or a feathers create action. in both cases, the components should be written such that when the state changes, the component's render function displays the state appropriately. this is the concept of "unidirectional data flow": state flows into the render functions and actions change state which causes a re-flow.

Comment (ahdinosaur) [inline app/highlights/components/newHighlightList.js]: does that help at all?

File: app/profile/components/profile.js [hunk 3]
```diff
@@ -0,0 +1,29 @@
+const h = require('vdux/element').default
+
+const Highlight = require('../../highlights/components/Highlight')
```
Comment (ahdinosaur) [inline app/profile/components/profile.js]: unless "within a module" (such as `profile`), always do "top-down" `require`s (as in `require('app/highlights/components/Highlight')`)

File: app/highlights/components/newHighlightList.js (hunk 2 above)
Comment (michael-smith-nz) [inline app/highlights/components/newHighlightList.js]: Makes sense, thanks. Just curious how I could do it. Will come find you in person

**Pull request closed without merging**